<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>672</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>672</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>672</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="photoLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 50%; 
color: white;
 </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountLineEdit">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>250</y>
     <width>385</width>
     <height>65</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>385</width>
     <height>65</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>385</width>
     <height>65</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Segoe Print&quot;;font-size: 22px;

border-radius: 10px;
padding: 10px;

</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>47</x>
     <y>340</y>
     <width>385</width>
     <height>65</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>385</width>
     <height>65</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>385</width>
     <height>65</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Segoe Print&quot;;font-size: 22px;


border-radius: 10px;
padding: 10px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>672</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>672</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>672</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
 {
	 background-color: transparent;
 	 color: black; 
	 border: none;
	font: 700 14pt &quot;Microsoft YaHei UI&quot;;
 }
 QPushButton:hover 
{ 
	background-color: red;
 }
QPushButton:pressed 
{
	color: darkgray; 
}
</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="agreeCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>已阅读并同意服务协议和QQ隐私保护指引</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>385</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 14pt &quot;Segoe Print&quot;;
background-color: rgb(156, 216, 255);
color: rgb(255, 255, 255);
border-radius: 10px;
padding: 10px;
}

QPushButton:pressed
{
	background-color: rgb(0, 129, 214);
	color:rgb(63, 160, 224);
}

</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="moreSelectionButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>610</y>
     <width>85</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; 
color: rgb(170, 0, 0);
border: none;
font: 10pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>更多选项</string>
   </property>
  </widget>
  <widget class="QPushButton" name="onTopButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
 {
	 background-color: transparent;
 	 color: black; 
	 border: none;
	font: 700 14pt &quot;Microsoft YaHei UI&quot;;
 }
 QPushButton:hover 
{ 
	background-color: red;
 }
QPushButton:pressed 
{
	color: darkgray; 
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>photoLabel</zorder>
  <zorder>accountLineEdit</zorder>
  <zorder>passwordLineEdit</zorder>
  <zorder>closeButton</zorder>
  <zorder>agreeCheckBox</zorder>
  <zorder>registerButton</zorder>
  <zorder>titleLabel</zorder>
  <zorder>moreSelectionButton</zorder>
  <zorder>onTopButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
